src/osd/OSD.cc

```diff
@@ -7137,7 +7137,7 @@ void OSD::_committed_osd_maps(epoch_t first, epoch_t last, MOSDMap *m)
 	if (service.is_preparing_to_stop() || service.is_stopping()) {
 	  service.got_stop_ack();
 	} else {
-	  clog->warn() << "map e" << osdmap->get_epoch()
+	  clog->warn() << "map e" << osdmap->get_down_at(whoami)
```

Comment:
could put:
```c
clog->warn() << "map e" << osdmap->get_epoch()
 		        << " wrongly marked me down at"
 		        << osdmap->get_down_at(whoami);
```
otherwise this log line would be missleading. because `osdmap->get_down_at(whoami)` is not the epoch of the osdmap being digested.